gensrc/proto/olap_file.proto

```diff
@@ -206,6 +206,7 @@ message ColumnPB {
     repeated ColumnPB children_columns = 17;
     repeated string children_column_names = 18;
     optional bool result_is_nullable = 19;
+    optional segment_v2.ColumnPathInfo column_path_info = 20;
```

Comment:
add comment to explain each extracted subcolumn of variant column will generate a column in be schema

Reply:
comment already in ColumnPathInfo